### DISCUSSION TITLE
Client GUI not working properly

### DISCUSSION THREAD

**Discussion by alexdamore** [apohl79/audiogridder#1137] [Help]
Hi All,

first of all sorry for my English.. I tried to find the same topic here but (hope I am not wrong) didn't find anything similar.
I am playing since 2 days with Audiogridder using my 2 mac devices as following:

- Mac Studio M1 Max 
- MacPro M1 Pro

I installed the beta version optimized for M1 on both devices and all it's connected by Ethernet 1Gb (cable type 5).
The connection works fine and I can open the plugins and instruments without any problem but once the plugin GUI is opened I cannot modify anything from the client then I am obliged to move on the slave mac to modify the parameters.
I try to make action like moving parameters , button, slider, fader etc.... but nothing works

I just tried to switch o Generic Editor and it seems works properly.

Do you have any idea on what could be the root cause of my problem ?

Thanks in advance for all your advices

**Comment by apohl79** [comment]
check if the server has the accessibility permission.
